reference to schema (#149)
jupyter/nbformat: docs/format_description.rst

The Notebook file format
========================

The official Jupyter Notebook format is defined with
`this JSON schema <https://github.com/jupyter/nbformat/blob/master/nbformat/v4/nbformat.v4.schema.json>`_,
which is used by Jupyter tools to validate notebooks.

> This page contains some general points about the notebook format.

Reviewer: ```suggestion
This page contains a human-readable description of the notebook format.
```

I think we can be a bit more optimistic now than 'some general points' :slightly_smiling_face:

Author: good point :-)
